src/css/themes/bootstrap/LESS/infragistics.theme.css

```diff
@@ -1011,6 +1011,10 @@ body .ui-tooltip {
   font-weight: normal;
   border: 0;
 }
+/* Bug #741 S.D. 20.04.2017 When bootstrap theme used with bootstrap css text in editor can't be seen when editor has focus*/
```

Comment:
Svilen was committed the change only in the .less and now I commit the generated CSS file